noetic/Dockerfile

```
@@ -0,0 +1,82 @@
+FROM ubuntu:focal
+
+# basic args
+ENV ROS_DISTRO=noetic
+ARG USERNAME=user
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Add the following labels
+LABEL org.opencontainers.image.description="Docker Development Container"
+LABEL org.opencontainers.image.title="ROS1Dev"
+LABEL org.opencontainers.image.vendor="Tarun R"
+LABEL org.opencontainers.image.source="https://github.com/rtarun1/DockerForROS2Development"
```

Since the images will be hosted on this repo, will suggest to change this 
```suggestion
LABEL org.opencontainers.image.source="https://github.com/soham2560/DockerForROS2Development"
```

---

Done,  refer [7289362](https://github.com/soham2560/DockerForROS2Development/pull/3/commits/728936217944cffb603b659eacb6ec8e3a0ff3d5)